WooCommerce/Classes/ViewModels/Order Details/OrderPaymentDetailsViewModel.swift

```diff
@@ -204,19 +184,6 @@ final class OrderPaymentDetailsViewModel {
 
         return NSLocalizedString("Discount", comment: "Discount label for payment view") + " (" + output + ")"
     }
-
-    /// Calculate the net amount after refunds
-    ///
-    private func calculateNetAmount() -> NSDecimalNumber? {
-        guard let orderTotal = currencyFormatter.convertToDecimal(order.total) else {
-            return .zero
-        }
-
-        let totalRefundedUseCase = TotalRefundedCalculationUseCase(order: order, currencyFormatter: currencyFormatter)
-        let refundTotal = totalRefundedUseCase.totalRefunded()
-
-        return orderTotal.adding(refundTotal)
-    }
```

Comment:
It's really nice that so much of this logic is removed, the view model is much clearer now 👍